runtime/src/bank.rs

```diff
@@ -2198,6 +2204,12 @@ impl Bank {
                         self.feature_set.clone(),
                     );
 
+                    transaction_logs.push(
+                        Rc::try_unwrap(log_collector.unwrap())
```

Comment:
This `unwrap()` is safe right now but it's generally a good idea to avoid it when possible

Reply:
Yeah, I'm aware of the dangers and it felt sketchy using it here. Do you have an alternative for this case?

Reply:
Maybe unwrap_or_default. I'll test this out.